proto/proto/responses.proto

```diff
@@ -42,9 +42,6 @@ message SyncStateResponse {
     // data needed to update the partial MMR from `block_num` to `block_header.block_num`
     mmr.MmrDelta mmr_delta = 3;
 
-    // Merkle path in the updated chain MMR to the block at `block_header.block_num`
-    merkle.MerklePath block_path = 4;
-
     // a list of account hashes updated after `block_num` but not after `block_header.block_num`
     repeated AccountHashUpdate accounts = 5;
```

Comment:
Similar comment here: should we use `block_num + 1` here? Also, should we say `request.block_num` (here and in other filed comments) - otherwise, I'm not sure it is clear that `block_num` is related to the request.

Reply:
updated